jupyter/internal/bootstrap-jupyter-ext.sh

```diff
@@ -13,15 +13,17 @@ sudo apt-get install -y git
 # > Command failed: /bin/bash -x -e
 # > /home/vagrant/miniconda-3.18.3/conda-bld/work/conda_build.sh
 # If directory doesn't exist, attempt to create it.
-if [ ! -d "~/.local/share/jupyter/" ]; then
+if [[ ! -d "~/.local/share/jupyter/" ]]
+then
```

Comment:
Just curious, do you currently have a particular shell style guide preference? We haven't quite been consistent ourselves in enforcing shell style, though we've been more strict in some subprojects than others (especially if produced initially within Google) and generally follow the [Google bash style guide](https://google.github.io/styleguide/shell.xml?showone=Loops#Loops)

For now I'm okay with some variation since we haven't really been providing guidance on style guide decisions yet and certainly no need to block this PR on updating style consistency on unrelated parts of the scripts, but as a heads up, in the near future we may go through and do some cleanup to improve style consistency between the init actions (which in this case would change back to "if [[ ! -d ... ]]; then" all on one line).

Let me know if such refactoring would be a potential pain point for you to re-merge into your forks and/or whether you'd prefer to update any pieces accordingly within this PR before I test and merge it.